### ISSUE TITLE
Reusable lsat

### ISSUE THREAD

**Issue by tobi-bams** [stakwork/sphinx-stack#48]
# List of changes that need to occur to achieve this

## sphinx-relay
- create an endpoint to get active lsat
- make an update to the existing updateLsat endpoint to accommodate update of lsat when expired.
- add a new column to the lsat table called status, to keep track of lsat status
All these have been implemented, here is the PR [https://github.com/stakwork/sphinx-relay/pull/439]

## sphinx-bridge
- create a getActiveLsat method to get active lsat from sphinx-relay
- create the updateLsat method to update lsat in sphinx-relay
This has been merged to sphinx-bridge already, here is the PR: [https://github.com/kevkevinpal/sphinx-bridge/pull/2]

## sphinx-mac
- create a method that handles the getLsat method passed from sphinx-bridge
- create a method that handles the updateLsat method passed from sphinx-bridge
This has been implemented, here is the PR [https://github.com/kevkevinpal/sphinx-mac/pull/2]

## sphinx-nav-fiber
- get active lsat from sphinx-relay through sphinx-bridge
- update lsat when marked as expired from the bolt-wall server.
All this has been implemented, here is the PR [https://github.com/stakwork/sphinx-nav-fiber/pull/41]

**Comment by kevkevinpal** [comment]
Looked over the relay changes and they look good

Opened a new PR to the stakwork/sphinx-bridge with the changes you made here

https://github.com/stakwork/sphinx-bridge/pull/4

**Comment by kevkevinpal** [comment]
also I think you forgot this change
## jarvis-boltwall
https://github.com/stakwork/jarvis-boltwall/pull/1/files

**Comment by tobi-bams** [comment]
> also I think you forgot this change
> 
> ## jarvis-boltwall
> https://github.com/stakwork/jarvis-boltwall/pull/1/files

Ops!!!......

That's true, thanks....

**Comment by kevkevinpal** [comment]
Users can still use older relay version and not have the reusable lsats feature but if a user is on the older sphinx-mac version the browser will crash if they try and use getLsat and updateLsat

Also the older relay users wont have the `status` column so when they do upgrade all those lsats will be marked expired

**Comment by kevkevinpal** [comment]
Merged sphinx-relay and sphinx-bridge

waiting on sphinx-mac to be merged and released before merging the sphinx-nav-fiber changes
